Packs/ApiModules/Scripts/DemistoClassApiModule/DemistoClassApiModule.py

```diff
@@ -9,19 +12,99 @@ class DemistoWrapper(Demisto):  # type:ignore  [name-defined] # noqa: F821 # pyl
             (E.g., self.results). """
 
         class DemistoScript(DemistoWrapper):
+            def info(self, msg):
+                if "script being called is" not in msg.lower():
+                    script = demisto.callingContext['context']['ScriptName']
+                    msg = "[{}] {}".format(script, msg)
+                    super(DemistoScript, self).info(msg)
+
+            def debug(self, msg):
+                if "script being called is" not in msg.lower():
+                    script = demisto.callingContext['context']['ScriptName']
+                    msg = "[{}] {}".format(script, msg)
+                    super(DemistoScript, self).debug(msg)
+
             def getFilePath(self, id):
-                self.debug("Getting path of file entry with ID {}".format(id))
-                return super(DemistoScript, self).getFilePath(id)
+                res = super(DemistoScript, self).getFilePath(id)
+                self.debug("File path of entry with ID [{}] is [{}]".format(id, json.dumps(res)))
+                return res
+
+            def executeCommand(self, command, args):
```

Comment:
What do we think on raising in case of an error? Now we are asking for people to check iserror themselves and it could save that.
Also could be good to add handling to debug files no an remove them from the list